Sources/StreamChatSwiftUI/DefaultViewFactory.swift

```diff
@@ -1151,6 +1151,12 @@ extension ViewFactory {
     ) -> some View {
         AddUsersView(loadedUserIds: options.loadedUsers.map(\.id), onUserTap: onUserTap)
     }
+    
+    public func makeAttachmentTextView(
+        message: ChatMessage
```

Comment:
We now follow an approach where we put an options class instead of the params directly. So this would be something like `AttachmentTextViewOptions`, with one property `message` (to enable adding new properties in the future).

Reply:
no, just public is enough - we would like to control the extensibility of these classes internally.